### DISCUSSION TITLE
Meshes for Furniture and Objects

### DISCUSSION THREAD

**Discussion by Kai-X-Org** [HumanSupportRobot/QandA#69] [Q&A]
We discovered that in the tmc_gazebo_worlds package, there are model sdf files to a variety of objects. However, we would really like files that are .obj or .stl that can be taken as meshes in our program so we can get precise shape information on those objects for simulation. Where can we find such mesh files? Thank you so much! And apologies for posting many questions at once.

**Comment by masudam** [comment]
I'm sorry, but the tmc_gazebo_worlds package only contains .sdf files.
